test/api/requests/main.test.js

```diff
@@ -48,6 +51,26 @@ describe('Main Site', () => {
       .catch(done);
     });
 
+    it('should display the number of builds from the past week', (done) => {
+      Build.destroy({ where: {} }).then(() => {
+        const promises = Array.from(Array(10).keys()).map((day) => {
+          const date = moment().subtract(day + 1, 'days');
+          return factory.build({ createdAt: date });
```

Comment:
Interesting that setting `createdAt` with a `moment` instead of a regular javascript `Date` works. A quick search into the `Sequelize` source code looks like it does this conversion itself under the hood.